Systems/c172p-engine.xml

```diff
@@ -62,7 +62,100 @@
             </function>
             <output>/fdm/jsbsim/propulsion/engine[1]/air-intake-impedance-factor</output>
         </fcs_function>
-        
+
+    </channel>
+
+    <channel name="damage">
+        <fcs_function name="/damage/rpm-over-redline">
+            <function>
+                <sum>
+                    <property>/engines/active-engine/rpm</property>
+                    <value>-2750</value>
+                </sum>
+            </function>
+            <clipto>
+                <min>0</min>
+                <max>10000</max>
+            </clipto>
+        </fcs_function>
+
+        <fcs_function name="/damage/damage-level">
+              <function>
+                <ifthen>
+                    <eq>
+                        <p>/engines/active-engine/damage_allowed</p>
```

Comment:
Can you change this to `damage-allowed`?

Reply:
To be consistent with the other properties (line 696-707 engine.xml), no.